node-cli/team.js

```diff
@@ -0,0 +1,53 @@
+const inquirer = require("inquirer");
+const Realm = require("realm");
+const output = require("./output");
+const users = require("./users");
+
+
+exports.getTeamMembers = async () => {
+  const user = users.getAuthedUser();
+  try {
+    const teamMembers = await user.functions.getMyTeamMembers();
+    output.result(JSON.stringify(teamMembers, null, 2));
+  }
+  catch (err) {
+    output.error(JSON.stringify(err));
+  }
+};  
+
+exports.addTeamMember = async () => {
+  let currentUser;
+  try {
+    output.header("*** ADD A TEAM MEMBER ***");
+    currentUser = users.getAuthedUser();
+    const member = await inquirer.prompt([
+      {
+        type: "input",
+        name: "email",
+        message: "What is the new team member's email address?",
+      },
+    ]);
+    let result = await currentUser.callFunction("addTeamMember", [member.email]);
+    output.result("The user was added to your team.");
+  } catch (err) {
+    output.error(JSON.stringify(err));
```

Comment:
nit: Can we add a formatted error message if the user doesn't exist? e.g. `` `No user exists with email ${member.email}` ``

![image](https://user-images.githubusercontent.com/15657698/93505563-08fba780-f8e9-11ea-81fd-0ae34e501b3b.png)

Reply:
Should I do the same for `removeTeamMember()`?

Reply:
If `removeTeamMember()` accepts an arbitrary email input then yes I think we should. Alternatively, we could have users select from a list of all project members in pretty much the same way as my example code for `selectProject()`